hackernews/autokitteh.yaml

```diff
@@ -0,0 +1,15 @@
+version: v1
```

Comment:
Add short description to the top of this file.

example from `confluence_to_slack`:
```
# This YAML file is a declarative manifest that describes the setup of
# an AutoKitteh project that announces new Confluence pages in Slack.
```